packages/components/psammead-social-embed/src/__snapshots__/index.test.jsx.snap

```diff
@@ -2,6 +2,54 @@
 
 exports[`AmpSocialEmbed should render a notice when the provider is unsupported 1`] = `
 .c0 {
+  position: relative;
+}
+
+.c1 {
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-weight: 700;
+  font-style: normal;
+  font-size: 0.875rem;
+  line-height: 1rem;
+  background-color: #FFFFFF;
+  border: 0.125rem solid #222222;
+  color: #222222;
+  display: block;
+  left: 0;
+  line-height: 1;
+  padding: 0.75rem;
+  position: absolute;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+  top: 0;
+  z-index: 10;
+}
+
+.c1:not(:focus):not(:active) {
+  position: absolute !important;
```

Comment:
do we defiantly need that !important?

Reply:
and the one below? What is this CSS and .c3 for?

Reply:
Yes, we need !important here to ensure this rule is never overridden.

Reply:
Yes !important will do that and that is the reason for my question, as users should be able to override all our styles if they so wish to load their own CSS... So I'm wondering exactly why it is that we need to apply that here? Why might it be overridden? I believe the CSS still works without that looking on storybook at the example?

Reply:
If the .c1 and .c3 CSS declarations are to achieve visually hidden CSS, if it's helpful this is the CSS that we usually use for that: https://github.com/bbc/psammead/blob/797d78a9f289d4e09daaabd1039566f0fa7a7666/packages/components/psammead-visually-hidden-text/src/index.jsx#L4

Reply:
I've refactored this to make use of `psammead-visually-hidden-text`'s styles.